### DISCUSSION TITLE
Tomo preview generation fails for superres dataset

### DISCUSSION THREAD

**Discussion by asarnow** [nextpyp/.github#80] [General]
The ImageMagick command to generate the webp previews of the tomogram reconstructions was failing for me on a certain dataset, it turns out the issue occurs with superres K3 movies. I binned by 4 during frame alignment and by 12 for reconstruction. The raw tilt, aligned tilt series, and tomo side projection previews are all fine.

```
024-03-06 22:31:43 [INFO] pyp/system/local_run.py:65 | /usr/bin/montage -resize 1440x1024 -geometry +0+0 -tile 14x DA32-2_ts76_flip.000.png [... all the rest of tilt PNGs ...] DA32-2_ts76_rec.webp
2024-03-06 22:35:13 [ERROR] pyp/system/local_run.py:72 | Error! Cannot encode picture as WebP
Error code: 5 (BAD_DIMENSION: Bad picture dimension. Maximum width and height allowed is 16383 pixels.)
montage: delegate failed `'cwebp' -quiet %Q '%i' -o '%o'' @ error/delegate.c/InvokeDelegate/1928.
```

**Comment by abartesaghi** [comment]
That looks like a limitation of the webp format itself. We will look for ways around it, thanks for reporting.

**Reply by abartesaghi** [reply]
This was addressed in version [0.6.4](https://nextpyp.app/files/pyp/0.6.4/docs/index.html).
